package.json

```diff
@@ -83,5 +84,6 @@
     "tslib": "^2.4.0",
     "typescript": "^4.7.4"
   },
-  "prettier": "@github/prettier-config"
+  "prettier": "@github/prettier-config",
+  "browserslist": "extends @github/browserslist-config"
```

Comment:
There's a browserslistrc file I added recently, could you delete that in favor of this one?

Reply:
Edit: saw that you did 😂